Dockerfile

```diff
@@ -44,4 +45,15 @@ RUN mkdir /cmake && cd /cmake \
     && tar xf cmake-3.21.0-linux-x86_64.tar.gz \
     && rm cmake-3.21.0-linux-x86_64.tar.gz
 
-ENV PATH="/cmake/cmake-3.21.0-linux-x86_64/bin:${PATH}"
\ No newline at end of file
+ENV PATH="/cmake/cmake-3.21.0-linux-x86_64/bin:${PATH}"
+
+RUN cd root && wget https://github.com/intel/llvm/archive/refs/tags/sycl-nightly/20210725.tar.gz \
+	    && tar -xf 20210725.tar.gz \
+            && cd llvm-sycl-nightly-20210725 \
+            && cd buildbot \
+	    && python3.7 configure.py && python3.7 compile.py
```

Comment:
Is python version that important?

Reply:
yes, configure and compile won't work on python versions <3.7 (or the threshold is lower, I don't remember exactly, but it is there)